src/sentry/snuba/discover.py

```diff
@@ -669,7 +695,11 @@ def top_events_timeseries(
         # so the result key is consistent
         translated_groupby.sort()
 
-        results = {}
+        results = (
+            {"Other": {"order": limit + 1, "data": other_result["data"]}}
```

Comment:
Is this going to cause problems if a series for some reason gets named `Other`? Should we think about prefixing the series names going forwards?

Reply:
in a follow up i'll change the prefix for the top-5 to include order or an invalid tag character to avoid this overlap